rasa/__main__.py

```diff
@@ -62,6 +63,8 @@ def create_argument_parser() -> argparse.ArgumentParser:
     export.add_subparser(subparsers, parents=parent_parsers)
     x.add_subparser(subparsers, parents=parent_parsers)
 
+    plugins.add_plugin_parser(parser, subparsers, parents=parent_parsers)
```

Comment:
do we also want to update the output of `rasa --help` to include information about plugins?

Reply:
I added the link to the documentation of plugins to the output. `rasa --help` now looks as follows:

```bash
❯ rasa --help
usage: rasa [-h] [--version] {init,run,shell,train,interactive,telemetry,test,visualize,data,export,x,plugins} ...

Rasa command line interface. Rasa allows you to build your own conversational assistants 🤖. The 'rasa' command allows you to easily run most common commands like creating a new bot, training or evaluating models.

positional arguments:
  {init,run,shell,train,interactive,telemetry,test,visualize,data,export,x,plugins}
                        Rasa commands
    init                Creates a new project, with example training data, actions, and config files.
    run                 Starts a Rasa server with your trained model.
    shell               Loads your trained model and lets you talk to your assistant on the command line.
    train               Trains a Rasa model using your NLU data and stories.
    interactive         Starts an interactive learning session to create new training data for a Rasa model by chatting.
    telemetry           Configuration of Rasa Open Source telemetry reporting.
    test                Tests Rasa models using your test NLU data and stories.
    visualize           Visualize stories.
    data                Utils for the Rasa training files.
    export              Export conversations using an event broker.
    x                   Starts the Rasa X interface.
    plugins             List available command-line plugins. See 'https://rasa.com/docs/rasa/command-line-interface#command-line-plugins' for more information.

optional arguments:
  -h, --help            show this help message and exit
  --version             Print installed Rasa version
```